spark/spark-tensorflow-distributor/tests/integration/set_spark_conf.py

```diff
@@ -0,0 +1,53 @@
+import argparse
```

Comment:
Curious what this file does? Would mind adding some description in the beginning of this file?